cmd/dexctl/command_config.go

```diff
@@ -81,10 +82,36 @@ func runGetConnectorConfigs(cmd *cobra.Command, args []string) int {
 	fmt.Printf("Found %d connector config(s)\n", len(cfgs))
 
 	for _, cfg := range cfgs {
-		fmt.Println()
-		fmt.Printf("ID:   %v\n", cfg.ConnectorID())
-		fmt.Printf("Type: %v\n", cfg.ConnectorType())
+		switch typedCFG := cfg.(type) {
```

Comment:
Existing  direct DB call returns ConnectorConfig and Admin API returns  interface{} .I could not find a way to convert interface{} to ConnectorConfig hence I am doing type switch and printing connector configs different ways based on the type.